neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -191,6 +192,10 @@ type Guest struct {
 	// +optional
 	Ports []Port `json:"ports,omitempty"`
 
+	// Logical clock value corresponding to the desired resources of the VM.
+	// +optional
+	DesiredLogicalTime *LogicalTime `json:"desiredLogicalTime,omitempty"`
```

Comment:
Still not a fan of the "desired logical time" names... WDYT about `DesiredAt` or equivalent? (see my prior comment on the PR)

(also not sure that "desired" is the right word for this either... I often find that writing an explanation of the semantics helps make it clear what the right words to describe things are -- maybe some documentation would help with both here?)

Reply:
> No, that would be `CurrentLogicalTime`

My thinking was that "desired at" would be short for "what time was the current state desired at?"

> This is analogous to the scaling target itself: we have "current memory size" and "desired memory size", and the system works toward current becoming equal to the desired.

I don't think the analogy holds up — the "current" time is not actually equal to the value of the clock; it's equal to the time at which the current resources were requested.

Unless I'm missing something?

> > variables like desiredClock or desiredLogicalTime -- IMO, this reads with desired as an adjective affecting the clock/logical time, which I guess is not what's intended.
>
> On the contrary, this is intended.

I think I didn't express this well before ­— I'm reading `CurrentLogicalTime` ≈ "logical time right now" (i.e. literally the clock right now) and `DesiredLogicalTime` ≈ "logical time that we desire" (i.e., we want the time to be X).

Part of the reason I keep suggesting `FooAt`-like names is because (a) they don't have this issue, and (b) there's already precedent in k8s for that naming convention (see e.g. `metadata.{createdAt,deletedAt}`)

Reply:
Resolved, as there isn't any logical time left.